### PR TITLE
[EPPETA-2] Initial system architecture document

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: docs/system-architecture.md
Comment (stephenfuqua) [inline docs/system-architecture.md:1]: Be sure to click the "..." and then "view file" just above this comment to see the diagrams displayed properly.

File: docs/system-architecture.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+## System Diagram
+
+In the following diagram, "FE" means "front end" and "BFF" means backend for
+frontend. Micro data store is a small database used to support the BFF. Each of
+the components is described in more detail below.
+
+```mermaid
+flowchart LR
+    A(fa:fa-users) -->|via browser| B["`EPPETA *FE*`"]
+    B -->|fa:fa-times-circle https| C[ODS-API]
```
Comment (stephenfuqua) [inline docs/system-architecture.md]: Interestingly, GitHub does not show the font-awesome integration from Mermaid (`fa:fa-times-circle`). Thus you can't see that there's a big X there. I should modify that. You also can't see that the first node starts with a users icon.

Comment (stephenfuqua) [reply docs/system-architecture.md]:
```suggestion
    B -->|X https| C[ODS-API]
```

File: docs/system-architecture.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+## System Diagram
+
+In the following diagram, "FE" means "front end" and "BFF" means backend for
+frontend. Micro data store is a small database used to support the BFF. Each of
+the components is described in more detail below.
+
+```mermaid
+flowchart LR
+    A(fa:fa-users) -->|via browser| B["`EPPETA *FE*`"]
```
Comment (stephenfuqua) [inline docs/system-architecture.md]:
```suggestion
    A(users) -->|via browser| B["`EPPETA *FE*`"]
```

File: docs/system-architecture.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    D --> E[("micro
+data
+store")]
+```
+
+## Frameworks and Technology Choices
+
+### Frontend
+
+- [React](https://react.dev/) in "plain JavaScript" (not Typescript, at least
```
Comment (StevenM0) [inline docs/system-architecture.md:25]: Will try setting up TypeScript project first

**Review by StevenM0 — APPROVED** [review]

File: docs/system-architecture.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+  initially).
```
Comment (stephenfuqua) [inline docs/system-architecture.md]:
```suggestion
- [React](https://react.dev/) in "plain JavaScript" or Typescript.
```

**Review by StevenM0 — APPROVED** [review]

**Pull request merged**